sdk/elixir/dagger.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "elixir-sdk",
   "sdk": "go",
-  "source": ".",
-  "engineVersion": "v0.11.9"
+  "include": ["."],
```

Comment:
This looks too broad. Should include only what's necessary for what the runtime module needs to do to generate the SDK dir in the module's sources.

Reply:
I try adding `exclude` but it didn't works for me. Even changing to add `!` to include (like in the commit ca96233a7fd6d49c1a9dce5dbc2b28b2fd8c8e94) is also not working. 😢

Reply:
Oh, I just realize that `include` option doesn't effect `Directory("..")` on https://github.com/dagger/dagger/pull/7667/files#diff-6665f7b190862df91665d328260eaf53cdfbe4ddaddbbfb52a290f531a8671cdR24. I start confusing now.

Reply:
> Oh, I just realize that `include` option doesn't effect `Directory("..")` on https://github.com/dagger/dagger/pull/7667/files#diff-6665f7b190862df91665d328260eaf53cdfbe4ddaddbbfb52a290f531a8671cdR24. I start confusing now.

It works for me:
```json
{
  "name": "elixir-sdk",
  "sdk": "go",
  "include": [
    "dagger_codegen",
    "lib",
    "mix.exs"
  ],
  "source": "runtime",
  "engineVersion": "v0.12.1"
}
```
Note: Remove `Terminal()` from `runtime/main.go` as you can add that via the CLI.

```console
$ dagger call sdksource-dir
_type: Directory
entries:
    - dagger.json
    - dagger_codegen
    - lib
    - mix.exs
    - runtime
```

Reply:
Yeah, at 771ed2cb2f265849f10fff20d769f7f1cb677547, I'm now at the same state as you run:

```yaml
_type: Directory
entries:
    - .formatter.exs
    - .gitignore
    - LICENSE
    - dagger.json
    - dagger_codegen
    - lib
    - mix.exs
    - mix.lock
    - runtime
```

But when I try `mkdir potato && dagger init --sdk=github.com/wingyplus/dagger/sdk/elixir@771ed2cb2f265849f10fff20d769f7f1cb677547 --source=.`, I still got all files unlike get from `sdksource-dir`:

```
$ ls -la dagger_sdk
total 144
drwxr-xr-x  15 thanabodee  staff    480 Jul 19 23:30 ./
drwxr-xr-x   7 thanabodee  staff    224 Jul 20 01:42 ../
drwxr-xr-x  38 thanabodee  staff   1216 Jul 20 01:36 .changes/
-rw-r--r--   1 thanabodee  staff   1200 Jul 20 01:36 .changie.yaml
-rw-r--r--   1 thanabodee  staff   8866 Jul 20 01:36 .credo.exs
-rw-r--r--   1 thanabodee  staff     97 Jul 20 01:36 .formatter.exs
-rw-r--r--   1 thanabodee  staff    666 Jul 20 01:36 .gitignore
-rw-r--r--   1 thanabodee  staff  19531 Jul 20 01:36 CHANGELOG.md
-rw-r--r--   1 thanabodee  staff  10758 Jul 20 01:36 LICENSE
-rw-r--r--   1 thanabodee  staff   1570 Jul 20 01:36 README.md
-rw-r--r--   1 thanabodee  staff    419 Jul 20 01:36 dagger.json
drwxr-xr-x   4 thanabodee  staff    128 Jul 19 23:30 lib/
-rw-r--r--   1 thanabodee  staff   1159 Jul 20 01:36 mix.exs
-rw-r--r--   1 thanabodee  staff   3170 Jul 20 01:36 mix.lock
drwxr-xr-x   5 thanabodee  staff    160 Jul 20 01:36 test/
```

That's what I get confuse what's going on here. T_T

Reply:
> Note: Remove Terminal() from runtime/main.go as you can add that via the CLI. 

I using it during debugging during call `dagger init` but yeah, removing it. 🙏

Reply:
From discussed in https://discord.com/channels/707636530424053791/1133501257051553845/1263913471326818385. I think I start to understand what's happens now. 🙏